libgadget/slotsmanager.h

```diff
@@ -46,6 +46,13 @@ struct bh_particle_data {
     /*******************************************************/
     double DragAccel[3];
     double DFAccel[3];
+    MyFloat FullGravAccel[3]; /* Short-range tree acceleration at the most recent timestep
```

Comment:
Does this include the PM / long range accel?

Reply:
It does not...I will rename to make that clearer.